src/components/Footer/index.tsx

```diff
@@ -1,146 +1,91 @@
+              </a>
+            </Link>
+            <Link href="https://www.youtube.com/channel/UCUdXQMrVjrMMWG5NOZFpHqQ">
+              <a
+                target="_blank"
+                className="grid place-items-center h-10 w-10 rounded-full"
+              >
+                <FontAwesomeIcon icon={faTwitch} size="lg" />
+              </a>
+            </Link>
+            <Link href="https://www.youtube.com/channel/UCUdXQMrVjrMMWG5NOZFpHqQ">
+              <a
+                target="_blank"
+                className="grid place-items-center h-10 w-10 rounded-full"
+              >
+                <FontAwesomeIcon icon={faGithubAlt} size="lg" />
+              </a>
+            </Link>
+            <Link href="https://www.youtube.com/channel/UCUdXQMrVjrMMWG5NOZFpHqQ">
+              <a
+                target="_blank"
+                className="grid place-items-center h-10 w-10 rounded-full"
+              >
+                <FontAwesomeIcon icon={faInstagram} size="lg"/>
+              </a>
+            </Link>
           </div>
-          {/*           <div className="w-full lg:w-6/12 px-4 py-5 md:py-0">
-            <div className="flex flex-wrap items-top mb-6">
-              <div className="w-full lg:w-4/12 px-4 ml-auto">
-                <ul className="list-unstyled">
-                  <li>
-                    <a
-                      className="text-gray-200 hover:text-gray-400 font-semibold block pb-2 text-sm"
-                      href="#"
-                    >
-                      Código de conducta
-                    </a>
-                  </li>
-                </ul>
-              </div>
-            </div>
-          </div> */}
         </div>
-      </div>
-      <div className="grid place-items-center bg-gray-900 w-full h-10">
-        <div className="text-sm text-gray-500 text-center">
-          Made with{' '}
-          <FontAwesomeIcon
-            className="text-red-700 animate-pulse"
-            icon={faHeart}
-            size="lg"
-          />{' '}
-          by{' '}
-          <a
-            target="_blank"
-            className="transition duration-500 ease-in-out text-primary hover:text-tertiary font-extrabold"
-            href="https://www.linkedin.com/in/agustinamar/"
-          >
-            AM
-          </a>
-          ,{' '}
-          <a
-            target="_blank"
-            className="transition duration-500 ease-in-out text-primary hover:text-tertiary font-extrabold"
-            href="https://www.linkedin.com/in/hercase/"
-          >
-            HC
-          </a>{' '}
-          &{' '}
-          <a
-            target="_blank"
-            className="w-2 transition duration-500 ease-in-out text-primary hover:text-tertiary font-extrabold"
-            href="https://www.linkedin.com/in/mbavio/"
-          >
-            MB
-          </a>
         </div>
-      </div>
+      <p className='font-light'>© Frontendcafé 2021</p>
```

Comment:
Not sure if it's `Frontendcafé` or `FrontendCafé`. @agusmar can you clarify?

Reply:
Also, can we make the `2021` to be a dynamic variable for the current year using `Date`?

Reply:
Hi, it's FrontendCafé. Thanks!